src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -124,6 +125,9 @@
     <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
       <Link>Common\System\PlatformDetection.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\PlatformDetection.Unix.cs" Condition="'$(TargetGroup)' != 'uap'">
```

Comment:
I would use $(TargetsWindows) as otherwise when reading this line, it looks like an obvious bug.